prompts.md

```
@@ -4,85 +4,96 @@ Puedes añadir adicionalmente la conversación completa como link o archivo adju
 
-1. [Descripción general del producto](#1-descripción-general-del-producto)
-2. [Arquitectura del sistema](#2-arquitectura-del-sistema)
-3. [Modelo de datos](#3-modelo-de-datos)
-4. [Especificación de la API](#4-especificación-de-la-api)
-5. [Historias de usuario](#5-historias-de-usuario)
-6. [Tickets de trabajo](#6-tickets-de-trabajo)
-7. [Pull requests](#7-pull-requests)
+- [Índice](#índice)
+- [1. Descripción general del producto](#1-descripción-general-del-producto)
+- [2. Arquitectura del Sistema](#2-arquitectura-del-sistema)
+  - [**2.1. Diagrama de arquitectura:**](#21-diagrama-de-arquitectura)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)